src/assessments/common/instructions-and-labels-note.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+import * as Markup from '../markup';
+
+export const InstructionsAndLabelsNotes = () => (
+    <Markup.Emphasis>
+        Note: For WCAG 2.1 compliance both visible instructions and labels must be programatically related to a widget. For WCAG 2.0
```

Comment:
added comma after "WCAG 2.1 compliance" and updated spelling of "programmatically"
```suggestion
        Note: For WCAG 2.1 compliance, both visible instructions and labels must be programmatically related to a widget. For WCAG 2.0
```